### PR TITLE
bare minimum nucleo F767 HW CI

### LINKED ISSUE #7033: bare minimum nucleo F767 HW CI
We need a happy HwCiNucleoF7 test suite for off the shelf nucle F767

### PR DESCRIPTION
**Pull request by FDSoftware**
- re-enabled HW CI F767 job as: db94786
- openocd target for the new HW CI target
- added engine_type VW_ABA to F767 target
- edited vw.cpp with higher ADC pins, as the default PC1-5 conflict with the ethernet module on F767

Tests fail because the signature of the .ini and the firmware were different vs the F4 .ini, variables created to be able to pass a .ini from the github job

resolves #7033

### CODE REVIEW

**Comment by rusefillc** [comment]
@FDSoftware let's hack less https://github.com/rusefi/rusefi/commit/22c8a3fa586b58275c27c371cf35b792984bd4db

File: .github/workflows/hw-ci/run_hw_ci.sh [hunk 1]
```diff
@@ -4,6 +4,7 @@
 # .github/workflows/hw-ci/run_hw_ci.sh com.rusefi.HwCiProteus
 
 HW_SUITE=$1
+SIMULATOR_INI_FILE=$2
```
Comment (rusefillc) [inline .github/workflows/hw-ci/run_hw_ci.sh:7]: how is this new SIMULATOR_INI_FILE used in run_hw_ci.sh?

Comment (FDSoftware) [reply .github/workflows/hw-ci/run_hw_ci.sh:7]: Sorry I didn't commit it, I'll fix it with rebase now, it's used in `java_console/io/src/main/java/com/rusefi/LocalIniFileProvider.java ` to override the hardcoded variable `(INI_FILE_FOR_SIMULATOR_ROOT_PATH)` for the ini file

File: firmware/config/engines/vw.cpp [hunk 2]
```diff
@@ -19,6 +19,12 @@ void setVwAba() {
 	setFrankensoConfiguration();
 	engineConfiguration->camInputs[0] = Gpio::E0; // a random unused pin needed for happy HW CI
 
+	engineConfiguration->tps1_1AdcChannel = EFI_ADC_32;
```
Comment (rusefillc) [inline firmware/config/engines/vw.cpp:22]: this block is confusing, please add code comment?

Comment (FDSoftware) [reply firmware/config/engines/vw.cpp:23]: added on [f4e001d](https://github.com/rusefi/rusefi/pull/7136/commits/f4e001d2946d5643bb368c296a03f6c0382ee91a)

**Pull request merged**